tests/Sulu/Component/HttpCache/EventSubscriber/ContentMapperSubscriberTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\HttpCache\EventListener;
+
+use Prophecy\PhpUnit\ProphecyTestCase;
+use Sulu\Component\HttpCache\EventSubscriber\ContentMapperSubscriber;
+use Sulu\Component\Content\Event\ContentNodeDeleteEvent;
+use Sulu\Component\Content\StructureInterface;
+use Sulu\Component\HttpCache\HandlerInterface;
+
+class ContentMapperSubscriberTest extends ProphecyTestCase
+{
+    /**
+     * @var ContentMapperSubscriber
+     */
+    private $subscriber;
+
+    /**
+     * @var HandlerInterface
```

Comment:
https://github.com/sulu-cmf/sulu/pull/653/files#diff-16ccae90fac80c28a8bfb96018750928R58 is a better example as this handler only implements one interface